integration_tests/mockApis/educationAndWorkPlanApi.ts

```diff
@@ -47,7 +47,7 @@ const stubGetInductionLongQuestionSet = (prisonNumber = 'G6115VJ'): SuperAgentRe
           updatedByDisplayName: 'Alex Smith',
           updatedAt: '2023-08-29T10:29:22.457',
           updatedAtPrison: 'MDI',
-          educationLevel: 'UNDERGRADUATE_DEGREE_AT_UNIVERSITY',
+          educationLevel: 'SECONDARY_SCHOOL_TOOK_EXAMS',
```

Comment:
These are the changes I've made to the stubs that return Inductions from the Induction API (Education & Work Plan / PLP API)
All I've done is change various values to match what the existing cypress tests are expecting. They stubbed values are still 100% valid short and long question set Inductions 👍